src/System.Drawing.Design.Primitives/src/System/Drawing/Design/ToolboxItem.cs

```diff
@@ -0,0 +1,967 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+namespace System.Drawing.Design {
+    using System.Configuration.Assemblies;
+    using System.Runtime.InteropServices;
+    using System.Runtime.Serialization;
+    using System.ComponentModel;
+    using System.ComponentModel.Design;
+    using System.Diagnostics;
+    using System;
+    using System.Reflection;
+    using System.Collections;
+    using System.Collections.Generic;
+    using Microsoft.Win32;
+    using System.Drawing;
+    using System.IO;
+    using System.Text;
+    using System.Security;
+    using System.Security.Permissions;
+    using System.Diagnostics.CodeAnalysis;
+    using System.Runtime.Versioning;
+
+
+    /// <summary>
+    /// Provides a base implementation of a toolbox item.
+    /// </summary>
+    [Serializable]
```

Comment:
Does this need to be serializable?  /cc @ViktorHofer

Reply:
That is right. Design scenarios and some of the run-time ( custom editors) require this.  I am removing  [Serializable] attribute for now and differing the discussion to later ( when winforms team get a perfect scenario that is completely blocked by not having this). Winforms team also expect AssemblyName to be serialized but again, We can bring that discussion as a separate topic. @Tanya-Solyanik